training/index.html

```diff
@@ -0,0 +1,75 @@
+                </p>
+            </div>
+        </div>
+        
+
+        <hr class="whitespace">
+        <div class="row column">
+            <p>
+                The OME team is keen to provide outreach workshops in the community to
+                demonstrate the features of our tools within imaging workflows. The training
+                materials we develop are freely available along with the image data and
+                setup procedures used.
+            </p>
+            <p>
+                Our workshops range in length from a 1-hour overview of OMERO up to a full day.
+                For more detailed coverage, a 2-day workshop can be used to provide an introduction
+                to OMERO on one day and more advanced analysis, developer or sysadmin features
+                on the next.
+            </p>
+        </div>
+
+        <div class="row">
+            <div class="small-12 medium-3 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-envelope-o fa-2x"></i>
+                        <h4>Contact us</h4>
+                        <p class="card-caption">Get in touch to discuss a workshop at your institution.</p>
+                        <a href="{{ site.baseurl }}/support/" class="btn-blue button small">Contact Us</a>
+                    </div>
+                </div>
+            </div>
+            <div class="small-12 medium-3 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-green icon-green fa fa-calendar-o fa-2x"></i>
+                        <h4>Workshops</h4>
+                        <p class="card-caption">Check our events page for upcoming workshops.</p>
+                        <a href="{{ site.baseurl }}/events/" class="btn-blue button small">Events</a>
+                    </div>
+                </div>
+            </div>
+            <div class="small-12 medium-3 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-blue icon-blue fa fa-file fa-2x"></i>
+                        <h4>Materials</h4>
+                        <p class="card-caption">Browse training materials from recent workshops.</p>
+                        <a href="https://github.com/ome/training-repos/" target="_blank" class="btn-blue button small">Browse</a>
+                    </div>
+                </div>
+            </div>
+            <div class="small-12 medium-3 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-graduation-cap fa-2x"></i>
+                        <h4>Run a workshop</h4>
+                        <p class="card-caption">Find information on how to run your own workshop.</p>
```

Comment:
I assume as soon as we have a second category of training, this text will explicitly mention "your own OMERO workshop"?

Reply:
I guess we can decide that at the time. Either add another link from here or link to a /training/setup/ 
Maybe that would make a better name for the current /training/omero/ page if we're likely to add other non-omero training materials to the page later. @jburel ??

Reply:
we will other material e.g. training/idr
I think the name ``/training/omero`` is fine

Reply:
> I think the name /training/omero is fine

Agreed. I was more thinking there would be other pages as opposed to putting everything on one.